Automattic/wp-calypso
# Seller experience: Replace Woo logo for a svg version

#### Changes proposed in this Pull Request

* Replace the WooCommerce logo for a svg version.

#### Testing instructions



* Access `start/setup-site/intent?siteSlug=<SITE SLUG>&flags=seller-experience` (which enables the `seller-experience` feature flag)
* Click the "Start Selling" button
* Click "Continue"
* Check the new view
![image](https://user-images.githubusercontent.com/3801502/150831004-563f84a9-64c2-4c6c-b5ef-e3b62920a70d.png)

* The WooCommerce logo should be sharp



Fixes #60518
